lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -87,6 +90,39 @@ class DataDownload
           @$reports_request_response.text data['status']
           $(".msg-confirm").css({"display":"block"})
 
+    @$list_issued_certificate_table_btn.click (e) =>
+      url = @$list_issued_certificate_table_btn.data 'endpoint'
```

Comment:
This is fine with me. If you are updating an existing feature it is okay to add the new capabilities in CoffeeScript. I tried to find a wiki page with this recommendation and the closest I could find was this:

https://openedx.atlassian.net/wiki/display/TNL/How+to+add+a+new+page+to+LMS+or+Studio#HowtoaddanewpagetoLMSorStudio-UseJavaScriptinsteadofCoffeeScript